source/features/follow-file-renames.tsx

```diff
@@ -8,14 +8,18 @@ import features from '../libs/features';
 import * as api from '../libs/api';
 import {getCleanPathname} from '../libs/utils';
 
+interface Content {
+	files: Iterable<Record<string, string>>;
+}
+
 // TODO: ensure that pages with a single commit aren't fetched twice (api.ts' cache should do it automatically, after #1783 is merged)
 async function findRename(
 	user: string,
 	repo: string,
 	lastCommitOnPage: string
-) {
+): Promise<Content> {
 	// API v4 doesn't support it: https://github.community/t5/GitHub-API-Development-and/What-is-the-corresponding-object-in-GraphQL-API-v4-for-patch/m-p/14502?collapse_discussion=true&filter=location&location=board:api&q=files%20changed%20commit&search_type=thread
-	return api.v3(`repos/${user}/${repo}/commits/${lastCommitOnPage}`);
+	return api.v3(`repos/${user}/${repo}/commits/${lastCommitOnPage}`) as Promise<Content>;
```

Comment:
I'm casting here, but in the future it might be nice to genericise the payload of `v3` and `v4` so we can do this instead:

```javascript
v3<Content>(`repos/${user}/${repo}/commits/${lastCommitOnPage}`) // returns Promise<Content>
```

Reply:
I think we should make it generic, but default the default generic parameter to `any` to make it flexible. Then we can use generics if we want or not, depending on the context.

Reply:
I don't think this solves anything, let's please leave it as `AnyObject`, I don't want infinitely-nested types like the ones I dropped here:

https://github.com/sindresorhus/refined-github/pull/1783/commits/9fbc1931169f5a5228bf86c1a8873c59c8aa0d98

Previous discussion: https://github.com/sindresorhus/refined-github/pull/1783#discussion_r259628430